source/_integrations/zwave_js.markdown

```diff
@@ -1016,6 +1016,24 @@ Switching does not require renaming your devices.
 7. Uninstall the official **Z-Wave JS** add-on.
    - You are asked if you want to delete the related data. Keep it if you think you might switch back to the **Z-Wave JS** add-on later.
 
+### How to migrate from one adapter to a new adapter using Z-Wave JS UI add-on?
```

Comment:
Should we make this more general and not talk about the add-on but the Z-Wave JS UI project? Users can run Z-Wave JS UI in different ways.

Reply:
Not necessarily. If you're not using the add-on you just need to visit the Z-Wave JS UI url in your browser.